README.md

```diff
@@ -1347,6 +1212,23 @@ The `event` name in both the event stream and the hook is optional, in which cas
 
 For Server-Sent Events to work, your server must support HTTP streaming. If you don't get SSE to work check if your deployment platform has support for it.
 
+Because SSE count towards the limit of HTTP connections per domain, the useEventSource hook keeps a global map of connections based on the provided URL and options, as long as they are the same, the hook will open a single SSE connection and share it between instances of the hook.
```

Comment:
```suggestion
Because SSE count towards the limit of HTTP connections per domain, the `useEventSource` hook keeps a global map of connections based on the provided URL and options. As long as they are the same, the hook will open a single SSE connection and share it between instances of the hook.
```